package/models.py

```diff
@@ -375,6 +380,24 @@ def fetch_pypi_data(self):
                 if self.pypi_downloads is None:
                     self.pypi_downloads = total_downloads
 
+                # get documents_url from pypi
+                if self.documentation_url is None:
+                    if info["project_urls"].get("Documentation"):
+                        self.documentation_url = info["project_urls"].get(
+                            "Documentation"
+                        )
+                    elif info.get("docs_url"):
```

Comment:
Could we check this field first?

Reply:
I have seen that it is null for all the packages, I have seen but the name suggests it should contain the docs URL. I could not find any documentation for fields of JSON API. Can you share if you have any?. We can remove it if you think.

Reply:
Interesting, Let's keep it for now.